### PR TITLE
Allow multiple notification targets

### PR DESCRIPTION
**Pull request by dkurepa**
https://github.com/dotnet/arcade-services/issues/3633

### CODE REVIEW

File: src/DotNet.Status.Web/DotNet.Status.Web/Controllers/AlertHookController.cs [hunk 1]
```diff
@@ -173,7 +173,7 @@ private NewIssue GenerateNewIssue(GrafanaNotification notification)
 
 [Go to rule]({notification.RuleUrl})
 
-@{options.NotificationTarget}, please investigate
+{string.Join(',', options.NotificationTargets.Select(target => $"@{target}"))}, please investigate
```
Comment (oleksandr-didyk) [inline src/DotNet.Status.Web/DotNet.Status.Web/Controllers/AlertHookController.cs:176]: Maybe adding a space to the delimiter would make it a bit more readable? Like - `, `

**Review by oleksandr-didyk — APPROVED** [review]

File: src/DotNet.Status.Web/DotNet.Status.Web/Controllers/AlertHookController.cs (hunk 1 above)
Comment (dkurepa) [reply src/DotNet.Status.Web/DotNet.Status.Web/Controllers/AlertHookController.cs]: fair point, thanks

**Review by oleksandr-didyk — APPROVED** [review]

**Review by pavel-purma — APPROVED** [review]

**Review by premun — APPROVED** [review]

File: src/DotNet.Status.Web/DotNet.Status.Web/Controllers/AlertHookController.cs [hunk 2]
```diff
@@ -173,7 +173,7 @@ private NewIssue GenerateNewIssue(GrafanaNotification notification)
 
 [Go to rule]({notification.RuleUrl})
 
-@{options.NotificationTarget}, please investigate
+{string.Join(", ", options.NotificationTargets.Select(target => $"@{target}"))}, please investigate
```
Comment (missymessa) [inline src/DotNet.Status.Web/DotNet.Status.Web/Controllers/AlertHookController.cs:176]: So every Grafana alert is going to ping dnceng and prodcon with this change?

**Review by garath — APPROVED** [review]

File: src/DotNet.Status.Web/DotNet.Status.Web/Controllers/AlertHookController.cs (hunk 2 above)
Comment (premun) [reply src/DotNet.Status.Web/DotNet.Status.Web/Controllers/AlertHookController.cs:176]: Our team members were recently removed from dnceng so we want keep being notified

Comment (missymessa) [reply src/DotNet.Status.Web/DotNet.Status.Web/Controllers/AlertHookController.cs:176]: That's going to be a lot of noise for your team when y'all don't own most of the services that are getting pings.

Comment (premun) [reply src/DotNet.Status.Web/DotNet.Status.Web/Controllers/AlertHookController.cs:176]: We already get quite a lot of noise (from helix-machines or helix-service for example) but this is part of FR so we should be on these.

**Review by AlitzelMendez — APPROVED** [review]

**Pull request merged**